<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>oknoNowegoWpisu</class>
 <widget class="QDialog" name="oknoNowegoWpisu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(42, 31, 48);
color: white;
background-color: rgb(42, 31, 48);
color: white;
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="TytulLabel">
     <property name="font">
      <font>
       <pointsize>36</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Dodaj nowy wpis</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="TrescWpisu">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>200</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid white;</string>
     </property>
     <property name="placeholderText">
      <string>Treść nowego wpisu</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="TrescOpcjaSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="KolorLayout">
     <item>
      <spacer name="KolorLewySpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="KolorLabel">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Kolor wpisu</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="Kolor">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>45</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">Border: 1px solid white</string>
       </property>
       <item>
        <property name="text">
         <string>Biały</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Żółty</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Zielony</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="KolorPrawySpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="ZdjecieLayout">
     <item>
      <spacer name="ZdjecieLewySpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="ZdjecieLabel">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Dodaj zdjęcie do wpisu</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="WybierzZdjecie">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>45</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">Border: 1px solid white;</string>
       </property>
       <property name="text">
        <string>Wybierz</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="ZdjeciePrawyLayout">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="RodzajWspomnieniLayout">
     <item>
      <spacer name="RodzajWspomnieniLewySpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="RodzajWspomnieniaLabel">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Rodzaj wspomnienia</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="RodzajWspomnienia">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>45</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid white;</string>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <item>
        <property name="text">
         <string>Smutne</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Neutralne</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Radosne</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="RodzajWspomnieniaPrawySpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="DataTimeLayout">
     <item>
      <spacer name="DataTimeLewySpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="DataTimeLabel">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Data i godzina dodania wpisu</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="DateTimeEdit">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>45</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">Border: 1px solid white</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="DataTimePrawySpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="OpcjeDodajAnulujSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="ButtonLayout">
     <item>
      <spacer name="ButtonLewySpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Anuluj">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>45</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">Border: 1px solid white;</string>
       </property>
       <property name="text">
        <string>Anuluj</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Dodaj">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>45</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">Border: 1px solid white;</string>
       </property>
       <property name="text">
        <string>Dodaj</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="ButtonPrawySpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
